src/styles/App.css

```diff
@@ -0,0 +1,107 @@
+* {
+  font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
+.MainBody {
+  width: 500px;
+  background: rgb(214, 126, 126);
+  display: grid;
+  grid-template-rows: auto auto auto;
+  gap: 1px;
+  margin: 1% 0 0 auto;
+}
```

Comment:
- **[OPTIONAL]** It looks like the wireframe, well done 👏🏽 For a better UI I recommend adding a background image to personalize your app. May you could see some styles in the [solutions](https://www.theodinproject.com/lessons/foundations-calculator) on The Odin Project. NOTE: You need to sign in to see them.